testsuite/tests/ui/search/test_account_search.py

```diff
@@ -58,20 +59,22 @@ def test_search_multiple_accounts(login, navigator, custom_account, request):
         - you search account by specific name it will return the correct one
     """
     name = blame(request, "")
-    params = [dict(org_name=f"{name}_name", username="username", email="email@anything.invalid", password="123456"),
-              dict(org_name=f"{name}_name2", username="username2", email="email2@anything.invalid", password="123456"),
-              dict(org_name=f"{name}", username="user", email="mail@anything.valid", password="123456"),
-              dict(org_name=blame(request, "RedHat"), username="random", email="random@random.random",
-                   password="123456")
+    params = [dict(org_name=f"{name}_name", username=blame(request, "username"),
```

Comment:
I don't think that this is necessary change and it would only complicate more things. @dhlavac @pehala opinion?

Reply:
By doing this ^^ you don't repeat the same exact code 4 times and it is obvious what names are your accounts using, thus reading this and asserts are clear

Reply:
Personally I think all irrelevant unrelated changes should be remove from this PR at first place and only "test for product search" code should be included. After that a discussion (and decision) should be little bit easier.

Reply:
@jsmolar I agree with you, it would be easier to read that test, and absolutely agree with @mganisin !!! It would be great to move unrelated changes to separate PR! (as I mentioned several times :) )